tests/garage/envs/test_normalized_gym.py

```diff
@@ -35,8 +45,18 @@ def test_unflatten(self):
         for i in range(10):
             env.reset()
             for e in range(5):
-                action = env.action_space.sample()
-                next_obs, reward, done, info = env.step(action)
+                a = env.action_space.sample()
+                a_copy = a if isinstance(a, (int, float)) else a.copy()
+                next_obs, reward, done, info = env.step(a)
+
+                # Check for side effects
+                if isinstance(a, (int, float)):
+                    assert a == a_copy,\
```

Comment:
This code does not test flattening and therefore does not belong in this test case. It is a separate test case (function). If the test cases share common code, use `setUp()`.